llvm/unittests/Support/KnownBitsTest.cpp

```diff
@@ -362,7 +362,7 @@ TEST(KnownBitsTest, BinaryExhaustive) {
         return KnownBits::absdiff(Known1, Known2);
       },
       [](const APInt &N1, const APInt &N2) {
-        return APIntOps::absdiff(N1, N2);
+        return APIntOps::abdu(N1, N2);
       },
```

Comment:
Do you need the lambda at all? Can the whole entry be just `APIntOps::abdu`?

Reply:
Yes, we can do that - the KnownBits function as well in many cases - although we've tended to keep the lambdas for all cases so far (smin/smax  etc. above) - I have no strong preference.

Reply:
Keeping the lambda seems silly if we don't need it, but we can always clean them all up later.